### PR TITLE
added dockerized whales-names

### PR DESCRIPTION
**Pull request by Rick-van-Dam**

### CODE REVIEW

File: boot.cmd [hunk 1]
```diff
@@ -0,0 +1 @@
+whales-names.cmd C:\driversetc\hosts
```
Comment (joostmeijles) [inline boot.cmd]: waarom is dit niet `C:\drivers\etc\hosts`

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+COPY package-lock.json package-lock.json
+
+RUN npm install
+RUN npm run build
+
+COPY boot.cmd boot.cmd
+
+FROM builder
+
+COPY --from=builder nodejs nodejs
```
Comment (joostmeijles) [inline Dockerfile]: dit is niet echt nodig toch? Zou verwachten dat 'ie het node-windows image ook in dit image gebruikt?

**Comment by joostmeijles** [comment]
Please add an example `docker-compose.yml` that shows how to use the Dockerized whales-names.

**Comment by Rick-van-Dam** [comment]
Optimized image size (now using stefanscherer/node-windows:pure and npm prune --production is run in the builder stage)

**Comment by joostmeijles** [comment]
> Please add an example `docker-compose.yml` that shows how to use the Dockerized whales-names.

@Barsonax  please add above.

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+version: '2.4'
+
+services:
+ whalesnames:
+  image: "whales-names"
+  isolation: process
```
Comment (joostmeijles) [inline docker-compose.yml:6]: remove process isolation as it not necessary?

File: DockerBuild.ps1 [hunk 4]
```diff
@@ -0,0 +1 @@
+docker build . --isolation process -t whales-names
```
Comment (joostmeijles) [inline DockerBuild.ps1:1]: remove process iso as it is not necessary

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+version: '2.4'
+
+services:
+ whalesnames:
+  image: "whales-names"
```
Comment (joostmeijles) [inline docker-compose.yml]: remove quotes

**Review by joostmeijles — APPROVED** [review]

File: DockerBuild.ps1 (hunk 4 above)
Comment (Rick-van-Dam) [reply DockerBuild.ps1:1]: I left this in so that ppl don't need hyper v, al they need is to update to windows 1809 which is probably already done

**Pull request closed without merging**